tmt/steps/prepare/artifact/providers/file.py

```diff
@@ -125,3 +126,15 @@ def _download_artifact(
             self.logger.info(f"Successfully downloaded: '{artifact.id}'.")
         except Exception as error:
             raise DownloadError(f"Failed to download '{artifact}'.") from error
+
+    def contribute_to_shared_repo(
+        self,
+        guest: Guest,
+        source_path: tmt.utils.Path,
+        shared_repo_dir: tmt.utils.Path,
+        exclude_patterns: Optional[list[tmt.utils.Pattern[str]]] = None,
+    ) -> None:
```

Comment:
Would be nice to have a docstring here as well. I guess it's omitted because the parent class already has it? Still I find it quite useful to always have the description at hand when reading the code.

Reply:
I recommended not to have that in order to avoid diverging docstrings. Does your IDE not show inheritted docstrings?